tt_metal/hw/inc/dev_msgs.h

```diff
@@ -87,6 +89,7 @@ struct kernel_config_msg_t {
     volatile uint8_t dispatch_core_y;
     volatile uint8_t exit_erisc_kernel;
     volatile uint8_t pad1;
```

Comment:
Is there a description of these fields anywhere?

Reply:
Nothing that I know of unfortunately.